### PR TITLE
Add advisory for slice-deque

### PR DESCRIPTION
**Pull request by gnzlbg**

### CODE REVIEW

File: crates/slice_deque/RUSTSEC-0000-0000.toml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+[advisory]
+# Identifier for the advisory (mandatory). Will be assigned a "RUSTSEC-YYYY-NNNN"
+# identifier e.g. RUSTSEC-2018-0001. Please use "RUSTSEC-0000-0000" in PRs.
```
Comment (tarcieri) [inline crates/slice_deque/RUSTSEC-0000-0000.toml]: Can you remove the comments? Thanks.

Comment (gnzlbg) [reply crates/slice_deque/RUSTSEC-0000-0000.toml]: Done.

**Comment by tarcieri** [comment]
Per #69, for some keyword precedent, there's this (`memory-exposure`): https://github.com/RustSec/advisory-db/blob/master/crates/smallvec/RUSTSEC-2018-0003.toml#L7

I'd probably suggest `memory-corruption` and `rce` as keywords here if you're interested in adding some. I'd originally wanted to surface these in https://rustsec.org but haven't had time yet.

**Comment by gnzlbg** [comment]
@tarcieri I've added those two keywords (hope I did that correctly)

File: crates/slice_deque/RUSTSEC-0000-0000.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "slice_deque"
```
Comment (tarcieri) [inline crates/slice_deque/RUSTSEC-0000-0000.toml]: I noticed this a bit too late: RustSec goes by the "package" name (i.e. what appears on crates.io and `Cargo.lock`) as opposed to the crate name.

Looking at your Cargo.toml, the package name appears to be `slice-deque` (hyphen, not underscore):

https://github.com/gnzlbg/slice_deque/blob/master/Cargo.toml

I can fix this up for you when I assign the RUSTSEC identifier for this advisory.

Comment (gnzlbg) [reply crates/slice_deque/RUSTSEC-0000-0000.toml]: oh I missed that!

> I can fix this up for you when I assign the RUSTSEC identifier for this advisory.

as you wish, I can also send a PR that fixes this !

Comment (tarcieri) [reply crates/slice_deque/RUSTSEC-0000-0000.toml]: #71 fixes it

**Pull request merged**